### PR TITLE
Remove gateway URLs from `receipt.json`

### LINKED ISSUE #825: Remove HTTP gateway endpoint URL from receipt.json
Documentation for https://github.com/api3dao/airnode/issues/1225

### PR DESCRIPTION
**Pull request by amarthadan**
Close #825

Feel free to use better wording. The point is that the gateway URLs are no longer in `receipt.json` but are displayed on the terminal at the end of the deployment.

### CODE REVIEW

**Review by wkande — COMMENTED** [review]
It is not clear to me how the heartbeatId is created and by whom?

**Comment by amarthadan** [comment]
> It is not clear to me how the heartbeatId is created and by whom?

It's the same thing as here https://docs.api3.org/airnode/v0.7/reference/deployment-files/receipt-json.html. That fields there if the heartbeat is enabled. It's read from the config file actually.
I just didn't want to leave it empty. But it can be empty as well.

File: docs/airnode/v0.8/grp-providers/guides/build-an-airnode/http-gateways.md [hunk 1]
```diff
@@ -107,11 +107,9 @@ using the HTTP signed data gateway or via RRP.
...
 
 A gateway URL is generated for each gateway (when enabled) when Airnode is
-deployed. You can obtain the URLs `api.httpGatewayUrl` and
-`api.httpSignedDataGatewayUrl` from the
-[receipt.json](../../../reference/deployment-files/receipt-json.md) file
-returned by the Airnode deployer. They are also available as part of the payload
-sent from Airnode's [heartbeat](./heartbeat.md) to your specified heartbeat URL.
+deployed. You can see the URLs displayed on your terminal at the end of the
+deployment. They are also available as part of the payload sent from Airnode's
+[heartbeat](./heartbeat.md) to your specified heartbeat URL.
```
Comment (wkande) [inline docs/airnode/v0.8/grp-providers/guides/build-an-airnode/http-gateways.md:112]: I will change the content here a little.

File: docs/airnode/v0.8/grp-providers/guides/build-an-airnode/http-gateways.md [hunk 2]
```diff
@@ -143,10 +141,8 @@ required as part of the CURL call.
 gateway requires that the `encodedParameters` be encoded using
 [Airnode ABI](../../../reference/specifications/airnode-abi-specifications.md).
 
-Replace `<gatewayUrl>` in the examples below with a URL from the `receipt.json`
-file using the `httpGatewayUrl` or `httpSignedDataGatewayUrl` field. The
-[receipt.json](../../../reference/deployment-files/receipt-json.md) file is
-created when you deploy an Airnode.
+Replace `<gatewayUrl>` in the examples below with a URL display at the end of
+your deployment.
```
Comment (wkande) [inline docs/airnode/v0.8/grp-providers/guides/build-an-airnode/http-gateways.md]: I will change the content here a little.

File: docs/airnode/v0.8/grp-providers/tutorial/quick-deploy-aws/README.md [hunk 3]
```diff
@@ -246,8 +246,8 @@ URL:
 
 <code style="overflow-wrap:break-word;">&#60;httpGatewayUrl>/0xf466b8feec41e9e50815e0c9dca4db1ff959637e564bb13fefa99e9f9f90453c</code>
 
-- `<httpGatewayUrl>`: The base URL to the gateway, found in the `receipts.json`
-  file. Update the placeholder in the CURL example below with its value.
+- `<httpGatewayUrl>`: The base URL to the gateway, displayed at the end of the
+  deployment. Update the placeholder in the CURL example below with its value.
```
Comment (wkande) [inline docs/airnode/v0.8/grp-providers/tutorial/quick-deploy-aws/README.md]: I will change the content here a little.

File: docs/airnode/v0.8/grp-providers/tutorial/quick-deploy-gcp/README.md [hunk 4]
```diff
@@ -257,8 +257,8 @@ URL:
 
 <code style="overflow-wrap:break-word;">&#60;httpGatewayUrl>/0xf466b8feec41e9e50815e0c9dca4db1ff959637e564bb13fefa99e9f9f90453c</code>
 
-- `<httpGatewayUrl>`: The base URL to the gateway, found in the `receipts.json`
-  file. Update the placeholder in the CURL example below with its value.
+- `<httpGatewayUrl>`: The base URL to the gateway, displayed at the end of the
+  deployment. Update the placeholder in the CURL example below with its value.
```
Comment (wkande) [inline docs/airnode/v0.8/grp-providers/tutorial/quick-deploy-gcp/README.md]: I will change the content here a little.

**Comment by wkande** [comment]
Made a few minor changes

**Pull request merged**